internal/services/serviceconnector/service_connector_app_service_resource_test.go

```diff
@@ -551,6 +560,16 @@ resource "azurerm_app_service_connection" "test" {
   authentication {
     type = "systemAssignedIdentity"
   }
+  scope = "default"
```

Comment:
This suggests that the scope value might possibly need a default setting? And will need some validation in the schema?

Reply:
This is an optional field, no default value is required. As service team said, no specific validation is required.

Reply:
is `""` a valid? ie should this at least not permit empty strings?

Reply:
@jiaweitao001 - could you address this comment?

Reply:
Thanks for reminding. Addressed.